examples/src/main/java/org/apache/pdfbox/examples/lucene/LucenePDFDocument.java

```diff
@@ -360,7 +361,10 @@ private void addContent(Document document, InputStream is, String documentLocati
             {
                 stripper = new PDFTextStripper();
             }
-            stripper.writeText(pdfDocument, writer);
+            try {
+                stripper.writeText(pdfDocument, writer);
+            } catch (final PdfTimeoutException e) {
```

Comment:
why is exception being swallowed

Reply:
We already discussed. Exception is only thrown on the production issue call path.

Kun Zhang [2:51 PM] 
Ash, do you want to populate the exception to all callers of the bad function? Or only populate it on the specific path where the problem happened

Ashutosh Singh [3:01 PM] 
you can start with the problem path